docs/features/software-templates/recoverable-tasks.md

```diff
@@ -0,0 +1,47 @@
+---
+id: recoverable-tasks
+title: Recoverable tasks
+description: How to configure your templates to make your running tasks recover after system restart
+---
+
+It can happen that you decided to restart your Backstage instance, for example, by deploying a new version of it.
+And if at this moment some tasks (templates being executed) were in progress, by default, the execution
+will be terminated. Till task janitor close stale tasks for you, you keep seeing the status of these tasks
+as "in progress", but in fact they are not active anymore.
+
+If you would like to recover your tasks automatically, at this version you'll have an option to restart your task.
+
+For that you have to specify in your template a recovery strategy:
+
+```yaml
+apiVersion: scaffolder.backstage.io/v1beta4
+kind: Template
+metadata:
+  name: my-recoverable-template
+spec:
+  recovery:
```

Comment:
Think it should be `EXPERIMENTAL_recovery` instead, then anything underneath it is treated as experimental.

Reply:
oh yeah, good catch, missed md file 🙈